comps/dataprep/src/Dockerfile

```diff
@@ -21,10 +21,11 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missin
     wget curl
 
 # Install ffmpeg static build
-RUN cd /root && wget https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-amd64-static.tar.xz && \
-mkdir ffmpeg-git-amd64-static && tar -xvf ffmpeg-git-amd64-static.tar.xz -C ffmpeg-git-amd64-static --strip-components 1 && \
-export PATH=/root/ffmpeg-git-amd64-static:$PATH && \
-cp /root/ffmpeg-git-amd64-static/ffmpeg /usr/local/bin/
+WORKDIR /root
+RUN wget https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-amd64-static.tar.xz && \
+    mkdir ffmpeg-git-amd64-static && tar -xvf ffmpeg-git-amd64-static.tar.xz -C ffmpeg-git-amd64-static --strip-components 1 && \
+    export PATH=/root/ffmpeg-git-amd64-static:$PATH && \
+    cp /root/ffmpeg-git-amd64-static/ffmpeg /usr/local/bin/
```

Comment:
Btw. I have a separate ticket about this: https://github.com/opea-project/GenAIComps/issues/1494